cmd/node/factory/structs.go

```diff
@@ -816,6 +817,19 @@ func ProcessComponentsFactory(args *processComponentsFactoryArgs) (*Process, err
 		return nil, err
 	}
 
+	txLogsProcessor := transactionLog.NewNilTxLogProcessor()
+	if args.coreComponents.config.TxLogsStorage.Enabled {
```

Comment:
this construct might be moved directly where the sc processor is created.

Reply:
I think this should be mandatory, not optional - so no Enabled / Disabled flag

Reply:
Done and done